### PR TITLE
BD-3330: Add FAQ for field-level encryption

### PR DESCRIPTION
**Pull request by lydia-xie**
### Summary

My PR...

### Related PRs, issues, or features (optional)

- Closes [BD-3330](https://jira.braze.com/browse/BD-3330)

### Feature release date (optional)

- N/A

### Contributor checklist

- [ ] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

**Review by tiffanyjlin — COMMENTED** [review]
Suggested response (feel free to reword):

No. Mail-to list-unsubscribe would send the plaintext decrypted email address to Braze. With field-level encryption enabled, we support the URL-based http: method, including one-click. We also recommend including a one-click unsubscribe link in the email body.

File: _docs/_hidden/private_betas/field_level_encryption.md [hunk 1]
```diff
@@ -113,3 +113,7 @@ The hashed email address is included in email delivery and engagement events.
 ### What email address should I expect to see in message archiving?
 
 The plaintext email address is included in messaging archiving. These are sent directly to the customer’s cloud storage provider and there may be other personal data included in the email bodies.
+
+### Can I use mail-to list-unsubscribe for subscription management with field-level encryption?
+
+No. Instead, we recommend using one-click and list-unsubscribe URL to process the unsubscribes from users.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/field_level_encryption.md:119]:
```suggestion
No. Mail-to list-unsubscribe would send the plaintext decrypted email address to Braze. With field-level encryption enabled, we support the URL-based HTTP: method, including one-click. We also recommend including a one-click unsubscribe link in your email body.
```

File: _docs/_hidden/private_betas/field_level_encryption.md [hunk 2]
```diff
@@ -113,3 +113,7 @@ The hashed email address is included in email delivery and engagement events.
 ### What email address should I expect to see in message archiving?
 
 The plaintext email address is included in messaging archiving. These are sent directly to the customer’s cloud storage provider and there may be other personal data included in the email bodies.
+
+### Can I use mail-to list-unsubscribe for subscription management with field-level encryption?
+
+No. Mail-to list-unsubscribe would send the plaintext decrypted email address to Braze. With field-level encryption enabled, we support the URL-based HTTP: method, including one-click. We also recommend including a one-click unsubscribe link in your email body.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/field_level_encryption.md:119]:
```suggestion
No. Using mail-to list-unsubscribe would send the plaintext decrypted email address to Braze. With field-level encryption enabled, we support the URL-based HTTP: method, including one-click. We also recommend including a one-click unsubscribe link in your email body.
```

File: _docs/_hidden/private_betas/field_level_encryption.md [hunk 3]
```diff
@@ -113,3 +113,7 @@ The hashed email address is included in email delivery and engagement events.
 ### What email address should I expect to see in message archiving?
 
 The plaintext email address is included in messaging archiving. These are sent directly to the customer’s cloud storage provider and there may be other personal data included in the email bodies.
+
+### Can I use mail-to list-unsubscribe for subscription management with field-level encryption?
+
+No. Using mail-to list-unsubscribe would send the plaintext decrypted email address to Braze. With field-level encryption enabled, we support the URL-based HTTP: method, including one-click. We also recommend including a one-click unsubscribe link in your email body.
```
Comment (rachel-feinberg) [inline _docs/_hidden/private_betas/field_level_encryption.md:119]:
```suggestion
No. Using mail-to list-unsubscribe will send the plaintext decrypted email address to Braze. With field-level encryption enabled, we support the URL-based HTTP: method, including one-click. We also recommend including a one-click unsubscribe link in your email body.
```

**Review by rachel-feinberg — APPROVED** [review]
Lgtm! Left a small suggestion

**Pull request merged**